pygfx/helpers/_gizmo.py

```diff
@@ -297,10 +326,19 @@ def _highlight(self, object=None):
     # %% Updating before each draw
 
     def update_matrix_world(self, *args, **kwargs):
-        """We overload this method, which gets called by the renderer
-        just before rendering. This allows us to prep the gizmo just
-        in time.
+        """Update the Gizmo's transform.
+
+        This method is overloaded from the base class to "attach" the gizmo to
+        the controlling object.
+
         """
+
+        # @almarklein: Assuming I interpreted above docstring correctly,
+        # wouldn't it be easier to make the gizmo a child of the controlled
+        # object and let the transform/linalg engine take care of the moving?
+        # (as its will move together with its parent, who's position the gizmo
+        # updates.)
```

Comment:
We tried that at the time, but decided against it. I can't recall the reasons though 🤔

Reply:
So we keep things as-is, or should we make an issue on attempting to change this (and shed some code in the process)?

Reply:
Let's keep it as-is. Tried to remember a bit harder. The thing is that if you make it a child of the object, you must undo the scaling of the world object, and depending on the mode, also the rest of the transform. This would make the calculations unnecessarily complex, defeating the purpose of using the parent-child relationship for selecting the object to control.